src/plugins/fullsize.ts

```diff
@@ -116,6 +116,12 @@ export function fullsize(editor: IViewWithToolbar) {
 			editor.container.classList.toggle('jodit_fullsize', condition);
 
 			if (editor.toolbar) {
+				if (!condition) {
```

Comment:
I don't know if theres a better solution. This is at least the easiest solution.